Marlin/ultralcd.cpp

```diff
@@ -3572,7 +3572,8 @@ void kill_screen(const char* lcd_msg) {
       #if EXTRUDERS > 1
         MENU_ITEM_EDIT(float52, MSG_CONTROL_RETRACT_RECOVER_SWAP, &swap_retract_recover_length, -100, 100);
       #endif
-      MENU_ITEM_EDIT(float3, MSG_CONTROL_RETRACT_RECOVERF, &retract_recover_feedrate_mm_s, 1, 999);
```

Comment:
Some units have the MSG_CONTROL_RETRACT_SWAPF includer  you have to write them . I'm writting directly from the website , I don't know how to synchronise with local code. It's why i suggest you make all this unit yourself

Reply:
That's fine. Strings not yet translated should be left out of language files so that we can automate tracking needed strings.